engine/runtime-instrument-runtime-server/src/main/java/org/enso/interpreter/instrument/runtime/server/RuntimeServerInstrument.java

```diff
@@ -108,7 +108,7 @@ public void onContextClosed(TruffleContext context) {}
   @Override
   protected void onCreate(Env env) {
     this.env = env;
-    env.registerService(this);
+
     if (TruffleOptions.AOT) {
```

Comment:
The usage of `TruffleOptions.AOT` is correct here as this is a Truffle instrument and it needs dependency on Truffle API.